cluster/router/condition/router.go

```diff
@@ -130,15 +166,15 @@ func (c *ConditionRouter) Route(invokers []protocol.Invoker, url common.URL, inv
 	if len(c.ThenCondition) == 0 {
 		return result
 	}
-	localIP, _ := gxnet.GetLocalIP()
 	for _, invoker := range invokers {
-		isMatchThen, err := c.MatchThen(invoker.GetUrl(), url)
+		invokerUrl := invoker.GetUrl()
+		isMatchThen, err := c.MatchThen(&invokerUrl, url)
 		if err != nil {
 			var urls []string
 			for _, invo := range invokers {
 				urls = append(urls, reflect.TypeOf(invo).String())
 			}
-			logger.Warnf("Failed to execute condition router rule: %s , invokers: [%s], cause: %v", c.Url.String(), strings.Join(urls, ","), err)
+			logger.Warnf("Failed to execute condition router rule: %s , invokers: [%s], cause: %v", c.url.String(), strings.Join(urls, ","), err)
```

Comment:
I think the invocation `c.MatchThen` returning non-nil error should be a serious problem. It means that maybe the codes have bugs, or the configuration has bug. It's different from `isMatchThen==false`.